compose/cli/utils.py

```diff
@@ -132,6 +133,7 @@ def get_version_info(scope):
     elif scope == 'full':
         return versioninfo + '\n' \
             + "docker-py version: %s\n" % docker_py_version \
-            + "%s version: %s" % (platform.python_implementation(), platform.python_version())
+            + "%s version: %s\n" % (platform.python_implementation(), platform.python_version()) \
+            + "OpenSSL version: %s" % ssl.OPENSSL_VERSION
```

Comment:
This broke python 2.6 compatiblity

`AttributeError: 'module' object has no attribute 'OPENSSL_VERSION' `

which I guess is ok, we just need to remove it from tox.ini